addons/sale_timesheet/models/account_move.py

```diff
@@ -87,7 +87,7 @@ def _timesheet_domain_get_invoiced_lines(self, sale_line_delivery):
     def unlink(self):
         move_line_read_group = self.env['account.move.line'].search_read([
             ('move_id.move_type', '=', 'out_invoice'),
-            ('move_id.state', '=', 'draft'),
+            ('parent_state', '=', 'draft'),
```

Comment:
In this case, that there's already another term that refers to move_id, I think this can be counter productive.

Reply:
I agree, will be fixed.